sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/DateTimeUtils.java

```diff
@@ -222,7 +223,7 @@ public static long safeMicrosToMillis(long micros) {
    * @return Unchanged timestamp sent for validation.
    */
   @SuppressWarnings("GoodTime")
-  public static Long validateTimestamp(Long ts) {
+  public static @CheckForNull Long validateTimestamp(@CheckForNull Long ts) {
```

Comment:
Reading spotbugs documentation, this is their recommendation, as `@Nullable` "doesn't do what you think it does". On the other hand, other tools all still use `@Nullable`. Since spotbugs doesn't seem to work for us any more, I will change these back to the more standard `@Nullable`.